website/docs/api/accessibility.md

```diff
@@ -4,11 +4,6 @@ title: Accessibility props
 sidebar_label: Accessibility
 ---
 
```

Comment:
why removed?

Reply:
> why removed?

Here is a reason -> https://github.com/MobileReality/react-native-select-pro/pull/59#issue-1205582035 And I confirm that this prop was removed few months ago but still existed in docs